packages/amplify-cli-npm/index.ts

```diff
@@ -16,5 +16,5 @@ export const install = async (): Promise<void> => {
   return binary.install();
 };
 
-// force minor version bump to 10.6
+// force minor version bump to 10.7.2
```

Comment:
No need to update this file unless it's for a minor or major bump

```suggestion
// force minor version bump to 10.6
```

Reply:
we should start using ascii art in those kind of comments instead of mentioning versions :D the content of comment is meaningless as long as there's any change inside.